templates/slide.html

```diff
@@ -17,19 +17,24 @@
 
 window.addEventListener( "pageshow", function ( event ) {
 
+    course = "{{course_name}}";
+    lec = "{{lec_name}}";
+    new_slide_name = course.concat("---").concat(lec);
     var historyTraversal = event.persisted ||
                          ( typeof window.performance != "undefined" &&
                               window.performance.navigation.type === 2 );
     if ( historyTraversal ) {
         logdata = JSON.stringify({
         action: 'cache_open',
+        middle: new_slide_name,
```

Comment:
change variable name "middle" to something meaningful, such as updated_slide_name or slide_name

Reply:
Changed middle to updated_slide in slide.html